src/plasma/plasma_store.cc

```diff
@@ -570,7 +572,9 @@ void delete_object(PlasmaStoreState *plasma_state, ObjectID object_id) {
   utarray_free(entry->clients);
   free(entry);
   /* Inform all subscribers that the object has been deleted. */
-  ObjectInfo notification = {.obj_id = object_id, .is_deletion = true};
+  ObjectInfo notification;
+  notification.obj_id = object_id;
+  notification.is_deletion = true;
```

Comment:
One problem with that kind of initialization (which I think I've run into in the past), is that if you change the struct definition it may still compile, but the values might correspond to different fields in the struct now.